### DISCUSSION TITLE
Is there a way to create a new project that supports R and Python?

### DISCUSSION THREAD

**Discussion by marcdotson** [posit-dev/positron#3814] [Q&A] [answered]
The project wizard is great for setting up new projects with reproducible environments for R or Python, but is there a way to systematically create a new project that includes reproducible environments for both R and Python?

I don't see an option to do this using the project wizard, so I'm assuming we have to manually do this by creating two projects and manually combining /renv and /.venv folders, or use {{renv}} to manage the Python dependencies -- but I want to make sure I'm not missing something!

**Comment by jmcphers** [comment, marked as answer]
You're correct, we have no tooling for this today. If you have a project that uses both R and Python and you want to manage dependencies for both of them, it's easiest to start by creating an R project with renv since (as you note) renv can manage both R and Python packages. More here:

https://cran.r-project.org/web/packages/renv/vignettes/python.html

**Reply by martinhulin** [reply]
This mixture of python/r projects becomes more and more common and also a more complex dependency manager is needed (like conda, as renv is not able to take care of system dependencies). So will conda or pixi be able to support r/python environments and will positron be able to find the intepreters (even a pure r projects under conda; eg. my company insists to have conda like environments as its the only way to install all needed dependencies)?
